cmd/deploy.go

```diff
@@ -92,6 +93,9 @@ func deployFn(_ *cobra.Command, _ []string) error {
 		),
 		clab.WithDebug(debug),
 	}
+
+	opts = PrepareExternalCAOptionEnviron(opts)
```

Comment:
looks like this should be called inside `certificateAuthoritySetup(c)`

Reply:
or maybe not... But then shouldn't the WithExternalCA option be invoked when settings.certificate-authority.cert and key are set?

Reply:
Thats a chicken-and-egg problem. The topo file is read by the clab instance. But the options are applied right after instantiation so the clab instance is already there and initialized.

Reply:
I removed the option entirely and set the env var values in the existing `certificateAuthoritySetup`